doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -41,14 +41,14 @@ Traceability to Reference Model.
 
 | Ref # | sub-category | Description | Traceability |
 |----|---------------|---------------------|---------------|
-| `req.gen.ost.01` | Open source | The Architecture **must** use OpenStack APIs.| |
-| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through OpenStack APIs. | |
-| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | |
+| `req.gen.ost.01` | Open source | The Architecture **must** use OpenStack APIs.| [RA-1 5.3](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#53-consolidated-set-of-apis) |
+| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through OpenStack APIs. | [RA-1 5.3](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#53-consolidated-set-of-apis) |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | OpenStack consists of both stateless and stateful services where the stateful services utilize a database. For latter see "[Configuring the stateful services](https://docs.openstack.org/ha-guide/control-plane-stateful.html)"|
 | `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. | |
-| `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. | |
+| `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. | Autoscaling is supported through Heat and also through Senlin, cluster management service. |
```

Comment:
Heat drives auto-scaling through monitoring/telemetry. But there are not much details in RA-1 yet about how heat should use monitoring for auto-scaling purpose. Neither we enforce that used monitoring solution should be able to trigger the auto-scaling.

Reply:
@deebhatia - yes I don't think we should recommend using Heat / ceilometer (gnocchi, etc.)  for the trigger as the application and VNFm is in a much better position to decide it needs more resource than relying on fragmented/deprecated OpenStack projects.   So we could suggest a webhook approach within the heat template?

Reply:
@iangardner22 If we use VNFm, it enforces that VNFm must be able to consume such webhooks. It is okay to have such restriction on VNFm, however, it should be documented as a **prerequisite**.

I found some more details on Heat's official [wiki](https://wiki.openstack.org/wiki/Heat/AutoScaling#Note). It says that autoscaling is being done by Senlin now and is obsolete in heat. Senlin has integration with Prometheus as a monitoring service. Should we recommend to have Senlin instead or stick with the heat's webhook approach?

Reply:
Difficult to recommend Senlin or another solution for autoscaling, it seems tooearly. As it is not mandatory, the topic can be moved into backlog for the next release

Reply:
https://github.com/cntt-n/CNTT/issues/1052
As suggested, logged an issue with 'backlog' label. @pgoyal01 Please remove the scalability tracing until the approach is final.

Reply:
@deebhatia. @iangardner22 and @karinesevilla: This is not a mandatory requirement. I am OK with it being addressed in a later release but I inserted the following in the 'Traceability' column:

>This requirement is currently not addressed but will likley be supported through [Senlin](https://docs.openstack.org/senlin/pike/index.html), cluster management service.

Reply:
@pgoyal01 I agree with your proposal